tg_bot/modules/weather.py

```diff
@@ -0,0 +1,74 @@
+import pyowm
+from pyowm import timeutils, exceptions
+
+from typing import Optional, List
+from tg_bot import dispatcher, updater, API_WEATHER
+from tg_bot.modules.disable import DisableAbleCommandHandler
+
+from telegram import Message, Chat, Update, Bot
+from telegram.ext import run_async
+
+@run_async
+def cuaca(bot: Bot, update: Update, args: List[str]):
+    location = " ".join(args)
+    if location.lower() == bot.first_name.lower():
+        update.effective_message.reply_text("I will keep an eye on both happy and sad times!")
+        bot.send_sticker(update.effective_chat.id, BAN_STICKER)
+        return
+
+    try:
+        bot.sendChatAction(update.effective_chat.id, "typing") # Bot typing before send message
+        owm = pyowm.OWM(API_WEATHER, language='en')
+        observation = owm.weather_at_place(location)
+        cuacanya = observation.get_weather()
+        obs = owm.weather_at_place(location)
+        lokasi = obs.get_location()
+        lokasinya = lokasi.get_name()
```

Comment:
Are the above lines guaranteed not to return `None`?
If yes, chain them up so there's only only necessary variable allocations. 
If no, add checks for that.